c2corg_ui/templates/helpers/edit.html

```diff
@@ -50,9 +50,14 @@ <h3 class="heading show-phone" translate>summary</h3>
       </div>
       % endif
       ## QUALITY
-      <div class="full comment" ng-class="{'has-success': saveCtrl.quality}" ng-init="qualityTypes = ${quality_types}">
-        <label translate>document_quality</label>
-        <select ng-options="type for type in qualityTypes" ng-model="saveCtrl.quality" class="form-control"><option value=""></option></select>
+      <div class="full comment" ng-class="{'has-success': saveCtrl.quality}">
+        <label translate>quality</label>
+        <select ng-model="saveCtrl.quality" class="form-control">
+          <option value=""></option>
+          % for q in quality_types:
```

Comment:
I had a similar problem (I think!?) with the preview modal where translations where not done as expected (ditto for dates converting with moment). It seems I have finally found another solution but I remember that @tsauerwein gave me a relevant tip, suggesting to use the parent scope or something like that, else ``mainCtrl`` was not available.